package main_test

import "testing"

func Test_JustFails(t *testing.T) {
	t.Errorf("failed to run the test")
}
